go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/refraction-networking/utls
+
+go 1.16
```

Comment:
Please lower the minimum version as low as possible.

Reply:
Sure! I suppose the minimum required version is Go 1.14 where `DialTLSContext` was added to `http.Transport`. (I am also using `log.Default`, which is Go >= 1.16, but that functionality is definitely not fundamental to this patch.)